src/NuGet.Core/NuGet.Protocol/Providers/PackageSearchResourceV3Provider.cs

```diff
@@ -23,9 +22,15 @@ public PackageSearchResourceV3Provider()
 
             if (serviceIndex != null)
             {
-                var rawSearch = await source.GetResourceAsync<RawSearchResourceV3>(token);
+                var endpoints = serviceIndex.GetServiceEntryUris(ServiceTypes.SearchQueryService);
```

Comment:
@nkolev92 @zivkan 
Is it ok this one doesn't create new PackageSearchResourceV3(rawSearch) anymore even though we have constructor for PackageSearchResourceV3?

Reply:
The provider is about the default experience. 
And we're saying the default experience doesn't use the raw search resource and that's ok. 

We're keeping the constructor there for binary compatibility.